scripts/package.ts

```diff
@@ -10,7 +10,7 @@ const installBindings = async () => {
 
   for (const platform of platforms) {
     await execute(
-      `./node_modules/.bin/node-pre-gyp install --directory=./node_modules/sqlite3 --target_platform=${platform} --target_arch=x64`,
+      `cross-env ./node_modules/.bin/node-pre-gyp install --directory=./node_modules/sqlite3 --target_platform=${platform} --target_arch=x64`,
```

Comment:
Pretty sure that calling cross-env like this requires having it installed globally.

Reply:
Also, since we don´t mention those bindings in the assets of pkg (in the main package.json), I think we use the bindings provided by the [native-extensions](https://github.com/botpress/studio/tree/master/packages/native-extensions) package instead.

Reply:
Nope, I don't have it installed globally. Dot commands doesn't work on windows. Using cross-env is the only safe option to run commands across all OS